File: scripts/task_tested_with_compile_script/regular-expressions/python/regular-expressions.py
import re

def executeTask(i):
    string = 'This is a matching string' + `i`
    re.search('string$', string)
    string = re.sub("string", "istyfied", string)
    return i

r = 1
for i in range(10000000):
	r = executeTask(i)

File: scripts/task_tested_with_compile_script/regular-expressions/perl/regular-expressions.pl
my $r = 1;

sub executeTask {
	my ($i) = @_;	
	$string = "this is a matching string" + $i;
	if ($string =~ /string$/) {}
	if ($string =~ s/ string/ istyfied/) {}
	return ($i );
}


for( $a=0; $a < 10000000; $a = $a + 1) {
	$r = executeTask($a );
}
